lib/utils/location.js

```diff
@@ -74,6 +78,9 @@ const appPageTypes: { [AppType]: {| default?: PageType, pageTypes: PageType[] |}
 	d2x: {
 		pageTypes: ['profile2x', 'profileCommentsPage'],
 	},
+	nmm: {
+		pageTypes: ['newModMail', 'composeNewModMail'],
```

Comment:
per https://github.com/honestbleeps/Reddit-Enhancement-Suite/pull/4790/files#r194815907

How about `modmail2x` and `modmail2xCompose`?